### ISSUE TITLE
Reproducing the paper result

### ISSUE THREAD

**Issue by Jongchan** [titu1994/keras-non-local-nets#4]
I wonder if you have successfully reproduced the numbers (accuracy) reported in the paper

**Comment by titu1994** [comment]
These networks are too large for me to train so cannot be trained by me.

**Comment by AliaksandrSiarohin** [comment]
Did you try with smaller dataset. Cifar10 for example, does it brings any improvement?
